tracing-attributes/src/lib.rs

```diff
@@ -258,6 +258,17 @@ mod expand;
 ///     42
 /// }
 /// ```
+/// 
+/// If the value being returned implements [`tracing::field::Value`], you can optionally record it
+/// in its raw form using `ret(Raw)`:
```

Comment:
nit, not a big deal: in general, we often avoid the use of second-person language ("you can ...") in API documentation. How about something like this:
```suggestion
/// If the value being returned implements [`tracing::field::Value`],  it can be recorded using its
/// `Value` implementation directly, using  `ret(Raw)`:
```